index.html

```diff
@@ -22,11 +28,14 @@ <h2>Experience</h2>
       </ul>
     </section>
 
-    <section id = "connect">
+    <section id = "connect" >
 <h2>Connect</h2>
-<a href="www.linkedin.com/in/farida-hyderi-a32700226</www.linkedin.com"> My linkedin </a>
-<a href="gmail.com"> gmail</a>
-<a href="instagram"> Instagram</a>
+
+
+<li><a href="www.linkedin.com/in/farida-hyderi-a32700226</www.linkedin.com"> My linkedin </a></li>
```

Comment:
This link could be improved so it correctly links to your LinkedIn page. You'll want something like the following format
```
<a href="[www.linkedin.com/in/farida-hyderi-a32700226](http://www.linkedin.com/in/farida-hyderi-a32700226)"> My LinkedIn </a>
```